snapshots/config.ts

```diff
@@ -0,0 +1 @@
+export const host = 'http://host.docker.internal:3000';
```

Comment:
@grafitto i think this is specific for docker for mac ? or is it working for you in linux also ? @Zasa-san ?

Reply:
The test lunch for me, they pass the step of connecting to Chromium. The error is `Execution context was destroyed, most likely because of a navigation.`.

Reply:
Removing this line or changing it for `http://localhost:3000` makes the tests throw the error `net::ERR_CONNECTION_REFUSED at http://localhost:3000`

Reply:
> @grafitto i think this is specific for docker for mac ? or is it working for you in linux also ? @Zasa-san ?

Its working well in linux